task/tkn-bundle-oci-ta/0.2/tkn-bundle-oci-ta.yaml

```diff
@@ -199,7 +199,7 @@ spec:
         task_version=$(yq '.metadata.labels."app.kubernetes.io/version"' "${FILES[@]}" | sed '/null/d' | tr -d '[:space:]')
         ANNOTATIONS+=("org.opencontainers.image.version=${task_version}")
 
-        task_dir=$(echo "${FILES[@]}" | awk -F'/' '{NF--; print $0}' OFS='/')
+        task_dir=$(echo "${FILES[0]}" | awk -F'/' '{NF--; print $0}' OFS='/')
```

Comment:
IIRC, this is being done because `task_dir` is used as the value for some of the annotations?
I understand the reasoning but it's a bit odd. Can we document it somewhere? Maybe in the README of this Task?

At some point, we may want to re-consider dropping the multiple tasks per bundle feature.

Reply:
@lcarva I added a comment in the script and updated the readme file.
Do you think something else is needed?

Reply:
Maybe in the README, it should list the annotations that are added and explain that the path of the first task definition is used to derive its value.

Not a blocker for this PR. Just a nice to have.